sim-lib/src/defined_activity.rs

```diff
@@ -49,7 +55,11 @@ impl PaymentGenerator for DefinedPaymentActivity {
                 "destination amount must not be set for defined activity generator".to_string(),
             ))
         } else {
-            Ok(self.amount)
+            let amount = self.amount.value();
+            if self.amount.is_range() {
+                log::debug!("Resampling payment amount. New value: {}", amount);
```

Comment:
I think that we can get away without these logs because we log the payment wait/amount at callsites?

Reply:
Yeah, I think it is pretty redundant atm. e.g.:

```
2024-04-05T10:00:17.332Z DEBUG [sim_lib::defined_activity] Resampling payment amount. New value: 216731
2024-04-05T10:00:17.332Z DEBUG [sim_lib] Generated random payment: alice(03628c...04b66e) -> bob(025e30...d26140): 216731 msat.
2024-04-05T10:00:17.332Z DEBUG [sim_lib::defined_activity] Resampling interval. New value: 7
2024-04-05T10:00:17.332Z DEBUG [sim_lib] Next payment for alice(03628c...04b66e) in 7s seconds.
```